ext/url/urlpattern.rs

```diff
@@ -0,0 +1,40 @@
+use deno_core::error::type_error;
+use deno_core::error::AnyError;
+
+use urlpattern::quirks;
+use urlpattern::quirks::MatchInput;
+use urlpattern::quirks::StringOrInit;
+use urlpattern::quirks::UrlPattern;
+
+pub fn op_urlpattern_parse(
+  _state: &mut deno_core::OpState,
+  input: StringOrInit,
+  base_url: Option<String>,
+) -> Result<UrlPattern, AnyError> {
+  let init = urlpattern::quirks::process_construct_pattern_input(
```

Comment:
missing unstable check

Reply:
ext/url isn't aware of unstable, and given this API is compute only, I think we can just expose the ops regardless of stability. (i.e. I am lazy and don't want to do the plumbing :smile:)